api/v1/models/TrackingEvent.js

```diff
@@ -17,4 +17,10 @@ TrackingEvent.findByName = function(searchName) {
     .fetch();
 };
 
+TrackingEvent.findAll = function() {
```

Comment:
I don't believe this is necessary

Reply:
Stats Service needs it to query all the tracked events and report the number of people who checked in.